components/serial-libs/openblas/SPECS/openblas.spec

```diff
@@ -10,12 +10,13 @@
 
 # OpenBLAS build that is dependent on compiler toolchain
 %define ohpc_compiler_dependent 1
+%define ohpc_uarch_dependent 1
 %include %{_sourcedir}/OHPC_macros
 
 # Base package name
 %define pname openblas
 
-Name:           %{pname}-%{compiler_family}%{PROJ_DELIM}
+Name:           %{pname}-%{compiler_family}%{UARCH_DELIM}%{PROJ_DELIM}
```

Comment:
Any chance to get UARCH_DELIM automatically added to PROJ_DELIM. If ohpc_uarch_dependent is set PROJ_DELIM is automatically extended with the UARCH_DELIM.